StockAnalyzerService/StockAnalyzerService/Program.cs

```diff
@@ -1,24 +1,52 @@
 using System;
 using System.Collections.Generic;
 using System.Linq;
+using System.Net.Http;
+using System.Net.Http.Headers;
 using System.Threading.Tasks;
 using Microsoft.Extensions.DependencyInjection;
 using Microsoft.Extensions.Hosting;
+using StockAnalyzerService.Service;
+using Serilog;
+using Microsoft.Extensions.Configuration;
+using System.IO;
 
-namespace StockAnalyzerService
-{
-    public class Program
-    {
-        public static void Main(string[] args)
-        {
+namespace StockAnalyzerService {
+    public class Program {
+        public static void Main(string[] args) {
             CreateHostBuilder(args).Build().Run();
         }
 
-        public static IHostBuilder CreateHostBuilder(string[] args) =>
-            Host.CreateDefaultBuilder(args)
-                .ConfigureServices((hostContext, services) =>
-                {
-                    services.AddHostedService<Worker>();
-                }).UseWindowsService();
+        public static IHostBuilder CreateHostBuilder(string[] args) {
+            var host = Host.CreateDefaultBuilder(args);
+            host = ConfigureLogging(host);
+            host = ConfigureServices(host);
+            return host.UseWindowsService();
+        }
+
+        public static IHostBuilder ConfigureLogging(IHostBuilder host) {
+            return host.ConfigureLogging(configureLogging => {
+                var configuration = new ConfigurationBuilder()
```

Comment:
can we extract this out? if we add additional information to the config file, we might need the configuration used in multiple places in the program.cs

Reply:
Done